python-threatexchange/threatexchange/fetcher/apis/tests/test_stopncii.py

```diff
@@ -25,27 +21,27 @@ def fetcher(api: StopNCIIAPI):
     return signal_exchange
 
 
-def test_fetch(fetcher: SignalExchangeAPI, monkeypatch: pytest.MonkeyPatch):
+def test_fetch(fetcher: StopNCIISignalExchangeAPI, monkeypatch: pytest.MonkeyPatch):
     monkeypatch.setattr("time.time", lambda: 10**8)
     collab = CollaborationConfigWithDefaults("Test")
     it = fetcher.fetch_iter([], collab, None)
-    delta = t.cast(SimpleFetchDelta, next(it))
+    delta = next(it)
 
-    assert delta.record_count() == 2
+    assert len(delta.updates) == 2
```

Comment:
Do you make actual API calls here? Can this test start failing because of the underlying database receiving updates?